javatests/arcs/showcase/imports/README.md

```diff
@@ -0,0 +1,4 @@
+# Import Showcase
+
+This showcase demonstrates the ability to import components of manifest files across directories.
+We'll demonstrate this usage through the example of a Tea import / export business.
```

Comment:
Maybe add a bit more here saying that particles are in one folder, schemas in another, etc. and that the ImportsTest.kt file runs the particles. Something like that.